plugin/mutate/source/dextool/plugin/mutate/backend/report/html/page_stats.d

```diff
@@ -61,13 +61,15 @@ void overallStat(const MutationStat s, Element base) {
             tuple("Total", s.total),
             tuple("Killed by compiler", cast(long) s.killedByCompiler),
             tuple("Skipped", s.skipped), tuple("Equivalent", s.equivalent),
-            tuple("Worklist", cast(long) s.worklist),
         ]) {
         tbl.appendRow(d[0], d[1]);
     }
-
-    base.addChild("p").appendHtml(
-            "<i>worklist</i> is the number of mutants that are in the queue to be tested/retested.");
+    {
+        auto foo = tbl.appendRow;
+        string infoText = "Worklist is the number of mutants that are in the same queue to be tested/retested";
+        foo.addChild("td").addChild("a", "Worklist").setAttribute("href", "worklist.html").addChild("div", "[?]").addClass("info-box").addChild("span", infoText).addClass("info-box-content");
```

Comment:
you probably want to create a helper function in `utility.d` for this code:
````d
.addChild("div", "[?]").addClass("info-box").addChild("span", infoText).addClass("info-box-content")
```

because it will be used at many places thus it is an advanted that it is easy to change.